README.md

```diff
@@ -1,2 +1,41 @@
-# HAASAlert
-CDOT solution to ingest HAASAlert data for emergency services data within GCP
+## Project Description
```

Comment:
Can we get a brief diagram of what the system looks like? where the websocket is hosted and the refresh token system?

Reply:
Yeah, I can work on that